<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homepage</class>
 <widget class="QMainWindow" name="homepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(1,35,38);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼모음T</family>
      <pointsize>25</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(82,242,226);</string>
    </property>
    <property name="text">
     <string>Simulator</string>
    </property>
   </widget>
   <widget class="QLabel" name="logolabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadMap">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>250</y>
      <width>300</width>
      <height>75</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>75</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕 ExtraBold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(82,242,226);
background-color: rgb(86,140,140);
border-radius: 35px;
border: 9px double rgb(82,242,226);</string>
    </property>
    <property name="text">
     <string>맵 파일 불러오기</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
